README_SOURCE-zh.md

```diff
@@ -0,0 +1,994 @@
+
+### **更新了什么？**
+
+:tada: _现在更新支持到 **TypeScript v3.7**_ :tada:
+:rocket: _升级到 `typesafe-actions@5.x.x` :rocket:
+
+<hr/><br/>
+
+</div>
+
+### **目标**
+
+- 完全的类型安全（支持 [`--strict`](https://www.typescriptlang.org/docs/handbook/compiler-options.html) 模式），并且在向应用的下游代码传递时，不会丢失类型信息（比如：缺少类型断言或用 `any` 来强行使用）
+- 使用高级 TypeScript 语言特性（诸如**类型推论**和**控制流分析**）来消除类型冗余、使类型声明简明扼要
+- 使用专门的 TypeScript [补充库](#complementary-libraries) 来减少类型代码的重复性和复杂度
+
+### **React、Redux、Typescript 生态系统**
+
+- [typesafe-actions](https://github.com/piotrwitek/typesafe-actions) - 为 Redux / Flux 架构中 "action-creators" 创造的类型安全实用工具集  
+- [utility-types](https://github.com/piotrwitek/utility-types) - TypeScript 常用泛型集合，能够补充 TS 自带的映射类型和别名 - 把它当成类型复用的 lodash。  
+- [react-redux-typescript-scripts](https://github.com/piotrwitek/react-redux-typescript-scripts) - 开发者工具配置文件，可用于遵循本指南的项目
+
+### **示例**
+
+- Todo-App playground: [Codesandbox](https://codesandbox.io/s/github/piotrwitek/typesafe-actions/tree/master/codesandbox)
+- React, Redux, TypeScript - RealWorld App: [Github](https://github.com/piotrwitek/react-redux-typescript-realworld-app) | [Demo](https://react-redux-typescript-realworld-app.netlify.com/)
+
+### **Playground 项目**
+
+[![Build Status](https://semaphoreci.com/api/v1/piotrekwitek/react-redux-typescript-guide/branches/master/shields_badge.svg)](https://semaphoreci.com/piotrekwitek/react-redux-typescript-guide)
+
+查看位于 `/playground` 文件夹中的 Playground 项目。它包含本指南所有的代码示例的源文件。它们都已使用最新版本的 TypeScript 和第三方类型定义包（诸如 `@types/react` 和 `@types/react-redux`）进行了测试，以确保示例是最新的，且没有随着类型定义升级而失效（基于 `create-react-app --typescript`）。
+> 我们创建了该 Playground 项目以便你可以简单地克隆到本地，并立即尝试本指南中所有的组件模式。它可以使你无需自己创建复杂的环境配置，直接在真实的项目环境中学习本指南的所有示例。
+
+## 贡献指南
+
+你能贡献并帮助改进本项目，如果你计划做出贡献，请查看我们的贡献指南：[CONTRIBUTING.md](/CONTRIBUTING.md)
+
+## 赞助
+
+你也能通过赞助 issues 提供帮助。
+通过 IssueHunt 平台进行赞助，bug 修复或功能请求之类的 issues 可以更快得到解决。
+
+我强烈建议你赞助自己期待解决的 issue，以便增加它的优先级并吸引贡献者解决。
+
+[![Let's fund issues in this repository](https://issuehunt.io/static/embed/issuehunt-button-v1.svg)](https://issuehunt.io/repos/76996763)
+
+---
+
+🌟 - _新内容及更新板块_
+
+## 目录
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [用 TypeScript 写 React & Redux - 完全指南](#%e7%94%a8-typescript-%e5%86%99-react--redux---%e5%ae%8c%e5%85%a8%e6%8c%87%e5%8d%97)
+    - [**更新了什么？**](#%e6%9b%b4%e6%96%b0%e4%ba%86%e4%bb%80%e4%b9%88)
+    - [**目标**](#%e7%9b%ae%e6%a0%87)
+    - [**React、Redux、Typescript 生态系统**](#reactreduxtypescript-%e7%94%9f%e6%80%81%e7%b3%bb%e7%bb%9f)
+    - [**示例**](#%e7%a4%ba%e4%be%8b)
+    - [**Playground 项目**](#playground-%e9%a1%b9%e7%9b%ae)
+  - [贡献指南](#%e8%b4%a1%e7%8c%ae%e6%8c%87%e5%8d%97)
+  - [赞助](#%e8%b5%9e%e5%8a%a9)
+  - [目录](#%e7%9b%ae%e5%bd%95)
+- [安装](#%e5%ae%89%e8%a3%85)
+    - [React & Redux 的类型定义](#react--redux-%e7%9a%84%e7%b1%bb%e5%9e%8b%e5%ae%9a%e4%b9%89)
+- [React - 类型定义速查表](#react---%e7%b1%bb%e5%9e%8b%e5%ae%9a%e4%b9%89%e9%80%9f%e6%9f%a5%e8%a1%a8)
```

Comment:
Here is a problem, this TOC is autogenerated using markdown headers because highlighted anchors don't have corresponding sections below they will be removed. You could fix this by adding markdown headers below and they will be added automatically by the script.

```doctoc --maxlevel=3 README_SOURCE.md README_SOURCE-zh.md```

you can find diff on my branch: https://github.com/piotrwitek/react-redux-typescript-guide/commit/f0c15a5494a38de856b21488977d3a935714730b


Basically the problems are that you should use Section headers as a source of truth NOT the table of contents which is autogenerated

Reply:
okay~
(I already found this problem days ago caused by my markdown editor plugins…)
I'd check that again with `doctoc` command is days ~